### ISSUE TITLE
GPKG format (binary package) support in eix

### ISSUE THREAD

**Issue by jeandestouches** [vaeth/eix#104]
Hello,
I recently switched to the new BINPKG_FORMAT=gpkg but eix only supports *.tbz2 or *.xpak (according to manpage)
Would you consider adding support for gpkg in eix ?
Thanks

**Comment by vaeth** [comment]
Fixed in eix-0.36.6

**Comment by zarMarco** [comment]
Sorry if I reopened this issue but I've installed eix.0.36.7 but if I launch eix --binary I don't see binary with gpkg

**Comment by vaeth** [comment]
Here it works. Are you sure that you have actually available versions?

**Comment by zarMarco** [comment]
Yes, I've installed some gpkg packages and I have setted file on /etc/portage/binhost

**Comment by vaeth** [comment]
This seems a strange directory to me: Is this what is shown with `eix --print PKGDIR`?

**Comment by zarMarco** [comment]
PKGBUILD is setted on /var/cache/binpkg. I followed this guide https://wiki.gentoo.org/wiki/Gentoo_Binary_Host_Quickstart for use binary and I installed a few binary. But if launch eix --binary I only saw packages on /var/cache/binpkg, so only packages installed with binary, but not packages that I never installed

**Comment by vaeth** [comment]
It seems that there is a misunderstanding: `PKGDIR` (that is, `/var/cache/binpkg`) are the binary packages, that is, which **can** be installed as binary - it does neither mean that they have been installed nor that they have been downloaded (they can get into this directory in many ways, including using quickpkg from a locally compiled packages), **This** is what eix is supposed to show. There is no plan to let eix query remote binhosts.

**Comment by zarMarco** [comment]
So if I understand right, my problem is with portage that doesn't populate folder on /var/cache/binpkg?

**Comment by vaeth** [comment]
> So if I understand right, my problem is with portage that doesn't populate folder on /var/cache/binpkg?

Yes. Maybe we can build a workaround in eix: Does /etc/portage/binhost have the same format as /var/cache/binpkg? In that case, it would be easy to add an optional path to eix which is treated the same way as `PKGDIR`. However, if the formats of these directories differ, I will probably not add such a support to eix.
